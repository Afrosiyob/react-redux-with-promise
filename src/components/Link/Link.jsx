import React from 'react'
import PropTypes from 'prop-types'

const Link = props => {

    const { title } = props


    console.log( title );


    return (
        <div>
            this is link
        </div>
    )
}

Link.propTypes = {
    title: PropTypes.string.isRequired
}

export default Link
