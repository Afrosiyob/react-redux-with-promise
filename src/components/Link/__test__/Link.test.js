import { render, screen } from "@testing-library/react";
import Link from "../Link";

import renderer from 'react-test-renderer';

describe( "link test describe", () => {
    it( "link test it", () => {
        const component = renderer.create( <Link title="this is title" /> )
        let tree = component.toJSON();
        expect( tree ).toMatchSnapshot();




    } )
} )