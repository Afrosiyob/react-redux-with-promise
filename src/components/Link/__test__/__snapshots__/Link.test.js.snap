// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link test describe link test it 1`] = `
<div>
  this is link
</div>
`;
